demisto_sdk/commands/common/hook_validations/readme.py

```diff
@@ -357,6 +357,24 @@ def is_context_different_in_yml(self) -> bool:
 
         return valid
 
+    def verify_demisto_in_readme_content(self):
+        """
+        Checks if there are the word 'Demisto' in the README content.
+
+        Return:
+            True if 'Demisto' does not exist in the README content, and False if it does.
+        """
+
+        is_valid = True
+
+        for line_num, line in enumerate(self.readme_content.split('\n')):
+            if 'demisto ' in line.lower() or ' demisto' in line.lower():
```

Comment:
let's gather all the lines which have the word in them and give one error back - not multiple times.

Reply:
done.